tests/parsing/parchmint-parser.test.js

```diff
@@ -833,1779 +833,1779 @@ describe('terminals', () => {
 
 //-- Begin Parchmint JSON strings --\\
 const readme_parchmint = require('./parchmints/readme_parchmint.json');
-const duplicates_readme_parchmint = require('./parchmints/duplicates_readme_parchmint.json');
```

Comment:
Should this `duplicates_readme_parchmint` requires be removed? Also I'm not sure what/why these includes are here, is it to include them for testing purposes? Could we just include them in the files that need them for testing instead or are they associated with these snippets?

Reply:
I removed that one because we're no longer using it in the tests. The ```readme_parchmint``` is a complete Parchmint file I included to test the ```parse``` method. They're not associated with the snippets, I just grouped all the test files/snippets together. Would you want me to move it somewhere else or add a comment to clarify?

Reply:
I think you should move the inclusion of the file to the test that its actually used in rather than including it into this mono-testing input file. I think another question might be do we gain anything more from having this file vs. the strings since we streamlined the parsing functionality in the last PR?

Reply:
I hadn't thought about that. Originally I used the file to also test whether the ```parse``` function could take an JSON object as a parameter. I'll paste the file contents as a template literal here to maintain consistency.